### DISCUSSION TITLE
--mmproj not available in Server?

### DISCUSSION THREAD

**Discussion by qnixsynapse** [ggml-org/llama.cpp#6610] [Q&A] [answered]
Can anyone show an example on how to send a request a server running a multi modal LLM like llava?

Here it seems the --mmproj argument is not being recognized. Has it been removed?

**Comment by phymbert** [comment, marked as answer]
Please see:
- #4216

Multimodal is not supported in the server ATM

**Comment by micoraweb** [comment]
Hello is it supported now ? :)

**Comment by gnusupport** [comment]
What about now?
